apps/embed-iframe/src/LoginModalContext.tsx

```diff
@@ -0,0 +1,31 @@
+import { useDisclosure } from "@chakra-ui/react";
+import React, { createContext, ReactNode, useContext, useState } from "react";
+
+interface LoginModalContextState {
+  isOpen: boolean;
+  onOpen: () => void;
+  onClose: () => void;
+  isLoading: boolean;
+  setIsLoading: (isLoading: boolean) => void;
+}
+
+const LoginModalContext = createContext<LoginModalContextState | undefined>(undefined);
+
+export const LoginModalProvider: React.FC<{ children: ReactNode }> = ({ children }) => {
```

Comment:
please don't use React.FC. it should've been caught by eslint. is it set up?

so, the definition should look like:
```typescript
export const LoginModalProvider = ({children}: PropsWithChildren) => {}
```